mp/src/game/client/momentum/ui/gameui/mainmenu/drawer/lobby/LobbyMembersPanel.cpp

```diff
@@ -302,6 +302,15 @@ void LobbyMembersPanel::OnContextTeleport(uint64 target)
     }
 }
 
+void LobbyMembersPanel::OnPersonaStateChange(PersonaStateChange_t *pParams)
+{
+    const auto pData = m_pMemberList->GetItem(FindItemIDForLobbyMember(pParams->m_ulSteamID));
```

Comment:
Same as the other callback, you should make sure the change flags here are for name only. There are [a ton of changes](https://partner.steamgames.com/doc/api/ISteamFriends#EPersonaChange) so it would be bad to constantly be doing this search for every friend on your friend list and lobby user.